install.js

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env node
 
-var execSync = require('child_process').execSync;
+const path = require('path');
+const execSync = require('child_process').execSync;
 console.log('Installing npm dependencies');
-execSync('npm install shelljs@0.7.0', {stdio:[0,1,2]});
+
+const pjson = require(path.join(__dirname, 'sfdx', 'package.json'));
```

Comment:
There are new dependencies now in package.json. So rather than maintain references in both locations. This guy will just install whatever is defined in package.json. This seems to be required based on the functionality demonstrated in the README file.